src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -95,11 +101,15 @@ static void DeleteOtherPlatformHost(Permutation permutation)
                 x86.Delete();
             }
         }
-
-        [TestFixtureSetUp]
-        public void RunBeforeAnyTests()
+    }
+    
+    [SetUpFixture]
+    public class TestSessionInitializer
+    {
+        [SetUp]
+        public void Setup()
         {
-            SessionId = Guid.NewGuid().ToString();
+            Base.SessionId = DateTime.UtcNow.Ticks.ToString();
```

Comment:
Why share the same session ID across *all* test? How can I filter on a single run?